### PR TITLE
EN-932 UserSearchForm unit tests updated

### PR DESCRIPTION
**Pull request by mcarta-rt**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

File: src/ui/users/list/UserListPage.js [hunk 1]
```diff
@@ -31,6 +32,11 @@ const ListUserPage = () => (
           />
         </Col>
       </Row>
+      <Row>
+        <Col md={6} mdOffset={3}>
```
Comment (alessioscalici) [inline src/ui/users/list/UserListPage.js]: Please replace md with xs as we are not handling a responsive layout

File: src/ui/users/list/UserSearchForm.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+import { reduxForm, Field } from 'redux-form';
+import { FormattedMessage } from 'react-intl';
+import { formattedText } from 'frontend-common-components';
+import { Row, Col, FormGroup, Button, InputGroup } from 'patternfly-react';
+import RenderRadioInput from 'ui/common/form/RenderRadioInput';
+
+const PROFILE_FILTER_OPTIONS = [
+  {
+    id: 'all',
+    label: 'All',
```
Comment (alessioscalici) [inline src/ui/users/list/UserSearchForm.js]: Please provide translations

File: src/ui/users/list/UserSearchForm.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+  onWillMount: PropTypes.func,
+  profileTypes: PropTypes.arrayOf(PropTypes.shape({
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    text: PropTypes.string,
+  })),
+};
+
+UserSearchFormBody.defaultProps = {
+  onWillMount: () => {},
+  setProfileType: () => {},
```
Comment (alessioscalici) [inline src/ui/users/list/UserSearchForm.js]: You could also set to null

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by alessioscalici** [comment]
retest this please

**Pull request merged**